Makefile

```diff
@@ -1,25 +1,34 @@
-
-k8s_setup:
-	kubectl apply -f kong-namespace.yaml
-	-./setup_certificate.sh
-
-run_cassandra: k8s_setup
-	kubectl apply -f cassandra-service.yaml
-	kubectl apply -f cassandra-statefulset.yaml
-	kubectl -n kong apply -f kong-control-plane-cassandra.yaml
-
-run_postgres: k8s_setup
-	kubectl -n kong apply -f postgres.yaml
-	kubectl -n kong apply -f kong-control-plane-postgres.yaml
-	kubectl -n kong apply -f kong-ingress-data-plane-postgres.yaml
-
-cleanup:
-	-kubectl delete -f cassandra-service.yaml
-	-kubectl delete -f cassandra-statefulset.yaml
-	-kubectl -n kong delete -f kong-control-plane-cassandra.yaml
-	-kubectl -n kong delete -f postgres.yaml
-	-kubectl -n kong dele 
-	-kubectl -n kong delete -f kong-ingress-data-plane-postgres.yaml
-	-kubectl certificate approve kong-control-plane.kong.svc
-	-kubectl delete csr kong-control-plane.kong.svc
-	kubectl delete -f kong-namespace.yaml
+
+k8s_setup:
+	kubectl apply -f kong-namespace.yaml
+	-./setup_certificate.sh
+
+run_dbless: k8s_setup
+	-kubectl create configmap kongdeclarative -n kong --from-file=declarative.yaml
+	kubectl create configmap kongdeclarative -n kong --from-file=declarative.yaml -o yaml --dry-run | kubectl replace -n kong -f -
```

Comment:
Why is this needed?
Isn't a kubectl apply sufficient here?

Reply:
I don't think it is no

```
kubectl apply configmap kongdeclarative -n kong --from-file=declarative.yaml
Error: unknown flag: --from-file
```

Reply:
Oh right, my bad.
Second question, why is the create being done twice? It Line 7 seems a bit weird, a left-over?

Reply:
Ah yeah it's kind of a hack (open to alternatives). It lets the make task succeed on first run and also let's one edit `declarative.yaml` and re-run the same task against an already running Kong dbless